querybook/webapp/hooks/dag/useExporterSettings.ts

```diff
@@ -23,7 +23,7 @@ function useExporterDataByName() {
         if (exporterNames.length === 0) {
             dispatch(fetchDAGExporters());
         }
-    }, [dispatch]);
+    }, [dispatch, exporterNames.length]);
```

Comment:
if there is no DAG exporter provided, this would infinite loop?

when `dispatch(receiveDAGExporters(exporters))`, we can probably not dispatch if there are no exporters